examples/riak/riaks.py

```diff
@@ -22,12 +22,19 @@
 
 from solar.interfaces.db import get_db
 
+from functools import partial
+
 from solar.events.controls import React, Dep
 from solar.events.api import add_event
 
 
 db = get_db()
 
+# use to separate riak cluster from everything else
+riak_tags = ['riak']
+# use to separate haproxy from everything else
+haproxy_tags = ['haproxy']
```

Comment:
You don't need this tag. All haproxy resources are already tagged with: "resources=haproxy"
https://github.com/Mirantis/solar/blob/master/resources/haproxy_service/meta.yaml#L15
https://github.com/Mirantis/solar/blob/master/resources/haproxy_service_config/meta.yaml#L23
https://github.com/Mirantis/solar/blob/master/resources/haproxy_config/meta.yaml#L21